src/App.vue

```diff
@@ -60,7 +60,9 @@
         </ul>
       </nav>
     </header>
-    <router-view/>
+    <div class="site-min-height">
```

Comment:
Please use a class with a more semantic name (i.e. `main-content` o `site-content.`), something that doesn't respond its styles.

Also, use `<main>` instead of `<div>`